toolbox/relnotes/hierarchy.go

```diff
@@ -0,0 +1,115 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"regexp"
+	"strconv"
+	"strings"
+
+	"github.com/google/go-github/github"
+)
+
+func hierarchicalNoteLayout(f *os.File, dict map[string]map[string]map[int]map[int]bool, issueMap map[int]*github.Issue) {
```

Comment:
This `dict` is crazy, you may want to split them, and create a structure for each level.

Reply:
@yutongz Updated.